### PR TITLE
Set poms to runtime. Add LogUtil.

### PR DESCRIPTION
**Pull request by dmitryintel**
* Some pom dependencies on logback-classic were redundant. Others could be left as runtime scope.
* Add back logging System.out  PrintStream: some initial log entries were lost without it.
* ServerControl can now easily use logback mechanism instead System.out.

### CODE REVIEW

File: osc-server/pom.xml [hunk 1]
```diff
@@ -165,13 +165,6 @@
             <version>${slf4j.version}</version>
             <scope>provided</scope>
         </dependency>
-         <dependency>
```
Comment (emanoelxavier) [inline osc-server/pom.xml]: It is strange that this is not needed here but it is needed in the control project. Should it be the other way around?

File: osc-control/src/main/java/org/osc/core/server/control/ServerControl.java [hunk 2]
```diff
@@ -110,8 +110,8 @@ public boolean isRunning() {
             } else if (cmd.hasOption("check")) {
                 // --check is used by vmidc.sh
                 if (isRunningServer()) {
-                    System.out.println(ServerControl.PRODUCT_NAME + " Server is already running.");
-                    System.out.println(getServerVersion());
+                    log.info(ServerControl.PRODUCT_NAME + " Server is already running.");
```
Comment (arvindn05) [inline osc-control/src/main/java/org/osc/core/server/control/ServerControl.java]: this should be sysout. If the server is running, then server control will not launch and its log is not setup...

File: osc-server/src/main/java/org/osc/core/broker/util/log/LogUtil.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+import java.io.PrintStream;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class LogUtil {
+
+    public static void initLog() {
```
Comment (arvindn05) [inline osc-server/src/main/java/org/osc/core/broker/util/log/LogUtil.java]: rename to redirectConsoleMessagesToLog

File: osc-server/pom.xml (hunk 1 above)
Comment (dmitryintel) [reply osc-server/pom.xml]: I actually observed it to fail without these dependencies. But with my recent change these dependencies are not needed.

**Pull request merged**